src/fu-tool.c

```diff
@@ -3041,6 +3050,14 @@ main(int argc, char *argv[])
 	     _("Filter with a set of device flags using a ~ prefix to "
 	       "exclude, e.g. 'internal,~needs-reboot'"),
 	     NULL},
+	    {"json",
+	     '\0',
+	     0,
+	     G_OPTION_ARG_NONE,
+	     &priv->as_json,
+	     /* TRANSLATORS: command line option */
+	     _("Output in JSON format"),
+	     NULL},
```

Comment:
this should probably split into a separate commit

Reply:
Fixed.